CONTRIBUTING.md

```diff
@@ -20,7 +20,10 @@
 
 1. [Open an issue here](https://github.com/NuGet/Home/issues) and get some feedback from the NuGet team.
 1. Follow the instructions in [Code](#code)
-1. Make your change, and add tests. Please name your branch `dev-<userid>-<very-short-title>`.
+1. Make your change. Please name your branch `dev-<userid>-<very-short-title>`.
+1. Add tests.
+    * [Testing in .NET](https://docs.microsoft.com/en-us/dotnet/core/testing/)
+    * [Testing tools in Visual Studio](https://docs.microsoft.com/visualstudio/test/?view=vs-2019)
```

Comment:
A brief description to guide test creation would be great:

```suggestion
1. Add tests covering your contribution. NuGet has unit and functional tests, written as xUnit theories and facts. 
    * [xUnit.net theories](https://xunit.net/docs/getting-started/netfx/visual-studio#write-first-theory)
    * [Testing in .NET](https://docs.microsoft.com/en-us/dotnet/core/testing/)
    * [Testing tools in Visual Studio](https://docs.microsoft.com/visualstudio/test/?view=vs-2019) 
```

Reply:
I do think a more fleshed out page would make sense... just was trying to minimize the change to the overview page.